tests/storage/business/test_watcher.py

```diff
@@ -136,6 +136,10 @@ def test_partial_scan(tmp_path: Path):
     c.mkdir(parents=True)
     (c / "study.antares").touch()
 
+    upgrade_folder = diese / "folder/upgrade_folder.tmp"
+    upgrade_folder.mkdir(parents=True)
+    (upgrade_folder / "study.antares").touch()
```

Comment:
You'll have to add tests after this:
1- Scan the `diese` folder
2- Capture the log to see what you wrote
3- Still have 1 study sync on disk.